server/src/main/scala/sbt/server/SbtClientHandler.scala

```diff
@@ -150,7 +150,7 @@ class SbtClientHandler(
           // Now clean out the request handler
           readLineRequests -= serial
           confirmRequests -= serial
-        case None => // TODO - error
+        case _ => // TODO - error
```

Comment:
The reasoning is to not get MatchErrors. If you fear fall-throughs, the TODO should be implemented.